service/src/main/java/com/epam/esm/service/dto/CertificateDto.java

```diff
@@ -0,0 +1,128 @@
+package com.epam.esm.service.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.Set;
+
+@JsonRootName(value = "certificate")
+public class CertificateDto extends EntityDto<Long, CertificateDto> implements Serializable {
+    private static final long serialVersionUID = 1144649892569462913L;
+
+    private String name;
+    private String description;
+    private Double price;
+    private Integer duration;
+    @JsonFormat(shape = JsonFormat.Shape.STRING)
+    private Instant createDate = Instant.now();
+    @JsonFormat(shape = JsonFormat.Shape.STRING)
+    private Instant lastUpdateDate = Instant.now();
+
+    private Set<TagDto> tags;
+
+    public CertificateDto() {
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Double getPrice() {
```

Comment:
Double or float not recommended for price field. https://stackoverflow.com/questions/12496555/java-best-type-to-hold-price

Reply:
Changed to BigDecimal